src/index.js

```diff
@@ -203,14 +203,14 @@ function shouldLoadFile(filename) {
  * @returns Babel plugin object with Program and CallExpression visitors
  */
 function addMetadata({ types: t }) {
+  if (!process.env.BABEL_TEST_METADATA) {
+    return {};
```

Comment:
why return an object here?

Reply:
what about returning false?

Reply:
The expected return is an object. I confirmed this by trying to return just `return;` in simple-app. The error I get is `[BABEL] unknown: Plugin/Preset did not return an object.` So, we should return an empty object in this case.